source/_posts/Docker-Swarm.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Docker Swarm, Traefik, Let's Encrypt en Portainer
+date: 2019-11-03 12:00:00
+tags:
+- Docker swarm
+- Let's Encrypt
+- Traefik
+- Portainer
+---
+
+Case omschrijving 
+---
+Niet lang geleden heb ik besloten om me eens goed te gaan verdiepen in Docker. Je hoort het overal en het wordt steeds populairder en ik wilde eens gaan kijken wat hier nu zo vet aan is. Ik ben begonnen om me te certificeren, ik heb een cursus gevolgd en mijn examen met succes behaald. Ik vind het altijd belangrijk om te weten hoe iets in elkaar zit en waarom het zo in elkaar zit. Mijn idee is om een of meerdere productie applicaties op een Docker Swarm cluster te gaan draaien. In deze blog ga ik laten zien hoe ik een goede infrastructuur bouw om de applicaties te hosten. 
```

Comment:
Behalve vet, en dat je er van hoort is het een alternatief voor je huidige manier van werken, namelijk: FaaS en PaaS in public cloud.